sklearn/utils/_param_validation.py

```diff
@@ -122,6 +122,8 @@ def make_constraint(constraint):
         return _Booleans()
     if isinstance(constraint, str) and constraint == "verbose":
         return _VerboseHelper()
+    if isinstance(constraint, str) and constraint == "missing_values":
```

Comment:
consistency with array-like, random_state, etc being singular
```suggestion
    if isinstance(constraint, str) and constraint == "missing_value":
```

Reply:
yep but the parameter names is `missing_values`. This would be weird:

```python
_parameter_constraints = {
    "missing_values": ["missing_value"],
    ...
}